SwanCustomEnvironments/swancustomenvironments/scripts/builders/accpy.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+# Libraries need to be installed and not linked, due to their dependencies
+if [ -n "${INSTALL_NXCALS}" ]; then
+    SPARKCONNECTOR="sparkconnector==$(python -c 'import sparkconnector; print(sparkconnector.__version__)')"
```

Comment:
I am now thinking that we install sparkconnector and monitor together with their dependencies, right? I remember this pulled some extra packages that maybe we didn't want, but if we install them with --no-deps then they crash. Is this correct? What were the missing packages when installing with --no-deps?

Reply:
When we install sparkmonitor and sparkconnector without their dependencies (--no-deps) they start crashing. Sparkconnector, which is the first to be called, starts complaining about swanportallocator module not being found and so on, package after package. We're talking about around 40 libraries where some of them are jupyter and jupyterlab related and others like argon2, Jinja2, jsonpointer, etc

Reply:
From private conversation: testing if the jupyterlab dependency of sparkconnector can be skipped -- jupyterlab is the dependency that pulls most of the packages.